examples/ibc/ics_test.go

```diff
@@ -31,64 +32,83 @@ func TestICS(t *testing.T) {
 		},
 	}
 
-	for _, tt := range tests {
-		tt := tt
-		testname := tt.name
+	icsVersions := []string{"v3.1.0", "v3.3.0", "v4.0.0"}
```

Comment:
is there a difference between v3.1 and v3.3? if not we can just do 


```suggestion
        icsVersions := []string{"v3.3.0", "v4.0.0"}
```

Reply:
The two are different, yeah, v3.3.0 is the first one to need an explicit transform when coming over from 3.1